server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/MultiTableDataNameNode.java

```diff
@@ -25,9 +26,10 @@
  */
 @Getter
 @ToString(callSuper = true)
-public class MultiTableDataNameNode extends VariableWithLevelNode {
-  private int occursTimes;
-  private UsageFormat usageFormat;
+@EqualsAndHashCode(callSuper = true)
+public class MultiTableDataNameNode extends VariableWithLevelNode implements UsageClause {
```

Comment:
Nice idea to use interfaces! Why I did not think about it when implemented `Variable::getFormattedDisplayLine`?